common/game_rules/zz_hfe_vanilla_replace_game_rules.txt

```diff
@@ -112,13 +112,14 @@ can_spawn_debris = {
 		or = {
 			is_country_type = default
 			is_country_type = hfe_pirate
+			is_country_type = human_fallen_empire
+			is_country_type = fallen_empire
```

Comment:
I do not see why fallen empires should loot debris.

Reply:
Okay, done